tests/hash-backend-integration/src/tests/subgraph/friendship.test.ts

```diff
@@ -189,136 +189,124 @@ afterAll(async () => {
 });
 
 describe("Ontology queries", () => {
-  it("read data types", async () => {
-    const resolve_depths = [
-      zeroedGraphResolveDepths,
-      {
-        ...zeroedGraphResolveDepths,
-        inheritsFrom: { outgoing: 1 },
-      },
-      {
-        ...zeroedGraphResolveDepths,
-        inheritsFrom: { outgoing: 255 },
+  it.each([
+    zeroedGraphResolveDepths,
+    {
+      ...zeroedGraphResolveDepths,
+      inheritsFrom: { outgoing: 1 },
+    },
+    {
+      ...zeroedGraphResolveDepths,
+      inheritsFrom: { outgoing: 255 },
+    },
+  ])("read data types %#", async (resolve_depths) => {
```

Comment:
I'd perhaps go for `%j` or `%o` or something rather than printing the index here, but happy that we have identifiable output at least

Reply:
Spoke about this offline, and we agreed that the full object notation would be _a little bit_ too long.

Reply:
(see this)
![image](https://github.com/hashintel/hash/assets/25749103/99a3220e-b7b6-4474-86f9-3390dfecaf72)

If indices prove unclear later on we might swap to adding some brief descriptions alongside the test cases or something